pkg/tsdb/tempo/grpc.go

```diff
@@ -74,6 +76,39 @@ func getDialOpts(settings backend.DataSourceInstanceSettings, opts httpclient.Op
 		dialOps = append(dialOps, grpc.WithTransportCredentials(insecure.NewCredentials()))
 	}
 
+	// The following code is required to make gRPC work with Grafana Cloud PDC
+	// (https://grafana.com/docs/grafana-cloud/connect-externally-hosted/private-data-source-connect/)
+	proxyClient, err := settings.ProxyClient(ctx)
+	if err != nil {
+		logger.Error("Error getting proxy client. The Tempo plugin will not use the secure socks proxy", "error", err)
```

Comment:
I am not sure about this error handling here. If we are in PDC environment and we get error here does it even make sense to continue? I would assume the queries won't work anyway so wonder why we don't propagate the errors up?